exceptions/exceptions.hh

```diff
@@ -153,6 +153,14 @@ struct mutation_write_timeout_exception : public read_write_timeout_exception {
     { }
 };
 
+// Generating view updates for a single client request can take a long time and might not finish before the timeout is
+// reached. In such case this exception is thrown.
+// "Generating a view update" means creating a view update and scheduling it to be sent later.
+// This exception isn't thrown if the sending timeouts, it's only concrened with generating.
+struct view_update_generation_timeout_exception : public request_timeout_exception {
```

Comment:
A thought (probably nothing needed to be done here):

I've been thinking (something that @kostja said made me think about it). What if we reach a timeout and stop the view deletions in the middle? We need to confirm that in this state, not only will the client deletion fail, it will also **not yet be applied** in the base table, so that if the client retries the base deletion it will continue to delete more rows and they won't remain as view ghost rows.
I think we're fine: The read-before-write that lists all the view rows that need to be deleted requires that we have no yet deleted the base partition. So we probably only send the base deletion after having completed all the view row deletions.